Formula/homeshick.rb

```diff
@@ -0,0 +1,68 @@
+class Homeshick < Formula
+  desc "Git dotfiles synchronizer written in bash"
+  homepage "https://github.com/andsens/homeshick"
+  url "https://github.com/andsens/homeshick/archive/1.0.0.tar.gz"
+  sha256 "8bd3c46f1cfd68d82d97fa72a68a07c966092c77f276d1335cb390b2ec6062bf"
+  head "https://github.com/andsens/homeshick.git"
+
+  option "with-fish", "Build fish bindings"
+  option "with-csh", "Build csh bindings"
+  option "without-bash", "Build without bash bindings"
+
+  def install
+    inreplace "bin/homeshick", /^homeshick=.*/, "homeshick=#{opt_prefix}"
+
+    prefix.install "lib"
```

Comment:
You'd make me a mildly happier human if you replaced the two unconditional `prefix.install`s with an array 😉

```ruby
prefix.install %w[ lib homeshick.sh ]
```
Or even just:
```ruby
prefix.install "lib", "homeshick.sh"
```